src/lib.rs

```diff
@@ -203,23 +203,60 @@ pub struct Metrics {
     pub strikeout_thickness: f32,
 }
 
-pub trait Rasterize {
-    /// Errors occurring in Rasterize methods.
-    type Err: ::std::error::Error + Send + Sync + 'static;
+/// Errors occuring when using the rasterizer.
+#[derive(Debug)]
+pub enum Error {
+    /// Couldn't find font matching description.
```

Comment:
```suggestion
    /// Unable to find a font matching the description.
```

`Couldn't` would work too, but I always feel like `Unable` just sounds a bit nicer? Either way the `a` and `the` would apply.